File: hc-mdm-dao/src/main/java/com/hc/scm/mdm/dao/entity/BasMajor.java
package com.hc.scm.mdm.dao.entity;

import java.util.Date;

/**
 * Description: 请写出类的用途
 * All rights Reserved, Designed By hc* Copyright:   Copyright(C) 2014-2015
 * Company:     Wonhigh.
 * @author:     user
 * @date:  2015-05-11 20:18:21
 * @version 1.0.0
 */
public class BasMajor extends BasMajorKey {
    /**
     * 
     */
    private Integer orderNo;

    /**
     * 
     */
    private String creator;

    /**
     * 
     */
    private Date createTime;

    /**
     * 
     */
    private String modifier;

    /**
     * 
     */
    private Date modifyTime;

    /**
     * 
     */
    private String remarks;

    /**
     * 
     * {@linkplain #orderNo}
     *
     * @return the value of bas_major.order_no
     */
    public Integer getOrderNo() {
        return orderNo;
    }

    /**
     * 
     * {@linkplain #orderNo}
     * @param orderNo the value for bas_major.order_no
     */
    public void setOrderNo(Integer orderNo) {
        this.orderNo = orderNo;
    }

    /**
     * 
     * {@linkplain #creator}
     *
     * @return the value of bas_major.creator
     */
    public String getCreator() {
        return creator;
    }

    /**
     * 
     * {@linkplain #creator}
     * @param creator the value for bas_major.creator
     */
    public void setCreator(String creator) {
        this.creator = creator;
    }

    /**
     * 
     * {@linkplain #createTime}
     *
     * @return the value of bas_major.create_time
     */
    public Date getCreateTime() {
        return createTime;
    }

    /**
     * 
     * {@linkplain #createTime}
     * @param createTime the value for bas_major.create_time
     */
    public void setCreateTime(Date createTime) {
        this.createTime = createTime;
    }

    /**
     * 
     * {@linkplain #modifier}
     *
     * @return the value of bas_major.modifier
     */
    public String getModifier() {
        return modifier;
    }

    /**
     * 
     * {@linkplain #modifier}
     * @param modifier the value for bas_major.modifier
     */
    public void setModifier(String modifier) {
        this.modifier = modifier;
    }

    /**
     * 
     * {@linkplain #modifyTime}
     *
     * @return the value of bas_major.modify_time
     */
    public Date getModifyTime() {
        return modifyTime;
    }

    /**
     * 
     * {@linkplain #modifyTime}
     * @param modifyTime the value for bas_major.modify_time
     */
    public void setModifyTime(Date modifyTime) {
        this.modifyTime = modifyTime;
    }

    /**
     * 
     * {@linkplain #remarks}
     *
     * @return the value of bas_major.remarks
     */
    public String getRemarks() {
        return remarks;
    }

    /**
     * 
     * {@linkplain #remarks}
     * @param remarks the value for bas_major.remarks
     */
    public void setRemarks(String remarks) {
        this.remarks = remarks;
    }
}

File: hc-mdm-web/src/main/webapp/resources/app/model/BasMajor.js
/**
 * Description: (app/model)
 * All rights Reserved, Designed By Hc
 * Copyright:   Copyright(C) 2014-2015
 * Company:     Wonhigh.
 * @author:     user
 * @date:  2015-05-11 20:18:21
 * @version 1.0.0
 */
Ext.define('Hc_mdm.model.BasMajor', {
    extend: 'Ext.data.Model',
    alias: 'model.basmajor',
     fields: [
         {name: 'categoryNo', text: '', type: 'string'},
         {name: 'categoryId', text: '', type: 'int'},
         {name: 'categoryName', text: '', type: 'string'},
         {name: 'materialType', text: '', type: 'int'},
         {name: 'orderNo', text: '', type: 'int'},
         {name: 'creator', text: '', type: 'string'},
         {name: 'createTime', text: '', type: 'string'},
         {name: 'modifier', text: '', type: 'string'},
         {name: 'modifyTime', text: '', type: 'date'},
         {name: 'remarks', text: '', type: 'string'}
         ],
    idProperty: 'appNo',
    identifier: 'negative'
  });